### ISSUE TITLE
[Effect Suggestion] Drunk Camera

### ISSUE THREAD

**Issue by PinballGH** [gta-chaos-mod/ChaosModV#3191]
**Name of effect**
Drunk Camera / Drunk Cameraman

**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [ ] Long
- [X] Short (30 seconds)
- [ ] No duration

**Effect Description**

Moves the camera in random directions for 30 seconds, makes it hard for the player to aim at someone :)

**Comment by bruhidksmhlol** [comment]
that already exists

**Comment by PinballGH** [comment]
> that already exists

Sorry for not noticing it even tough I searched for quite a while. Could you link the effect in question please ?
The only one I see you might be talking about is this one : https://github.com/gta-chaos-mod/ChaosModV/issues/1748

Which does that, far from what I've suggested : https://youtu.be/GGzwrjjgwOk?t=142

**Comment by SeanMirrsen** [comment]
> > that already exists
> 
> Sorry for not noticing it even tough I searched for quite a while. Could you link the effect in question please ? The only one I see you might be talking about is this one because we happened to use the same effect name : #1748
> 
> Which does that : https://youtu.be/GGzwrjjgwOk?t=142
> 
> IMO, far from what I've suggested as it just moves in a rather circular way while mine is completely random and impossible to predict.

No, as in it literally already exists. The current "Drunk" effect, aside from messing with vehicle controls, also includes the swaying camera movement. As does the "I Feel Sick" effect.
